cmd/agent/subcommands/launchgui/command.go

```diff
@@ -86,7 +86,7 @@ func launchGui(config config.Component, cliParams *cliParams) error {
 	}
 
 	// Open the GUI in a browser, passing the authorization tokens as parameters
-	err = open("http://127.0.0.1:" + guiPort + "/authenticate?authToken=" + authToken + ";csrf=" + string(csrfToken))
+	err = open("http://127.0.0.1:" + guiPort + "/authenticate?authToken=" + authToken + "?csrf=" + string(csrfToken))
```

Comment:
Hmm... If we are using the new spec, I think the second (and later) params should maybe done with an ampersand instead of a double `?`:
```
http://127.0.0.1/authenticate?authToken=something&csrf=something
```

Then again, the code is old and I'm not 100% sure why we did the manual split of the params originally.

Reply:
@sgnn7 Thank you for the comment!
Replaced second `?` with `&` according to https://github.com/golang/go/issues/25192#issuecomment-385692432, https://github.com/DataDog/datadog-agent/pull/15636/commits/7abb8b295da69da7f128f4b8fc3dc39eb3e3aa04